ddtrace/tracer/tracer.go

```diff
@@ -130,7 +132,11 @@ const payloadQueueSize = 1000
 
 func newUnstartedTracer(opts ...StartOption) *tracer {
 	c := newConfig(opts...)
-	return &tracer{
+	envRules, _ := samplingRulesFromEnv()
```

Comment:
Why are we ignoring the error here? The user should know right?

Reply:
The reason we're ignoring it is that the `samplingRulesFromEnv` function already prints `WARN` messages about the errors it encounters. If you think it's better, we can eliminate that logging, and report the final error here instead.